modules/runner/src/integrationTest/java/org/apache/ignite/internal/BaseIgniteRestartTest.java

```diff
@@ -241,32 +236,17 @@ public static PartialNode partialNode(
     ) {
         AtomicLong lastRevision = new AtomicLong();
 
-        Consumer<Long> revisionCallback0 = rev -> {
-            if (revisionCallback != null) {
-                revisionCallback.accept(rev);
-            }
-
-            lastRevision.set(rev);
-        };
-
-        CompletableFuture<Void> configurationCatchUpFuture = RecoveryCompletionFutureFactory.create(
-                clusterCfgMgr,
-                fut -> new TestConfigurationCatchUpListener(cfgStorage, fut, revisionCallback0)
-        );
-
         CompletableFuture<?> startFuture = CompletableFuture.allOf(
                 nodeCfgMgr.configurationRegistry().notifyCurrentConfigurationListeners(),
                 clusterConfigRegistry.notifyCurrentConfigurationListeners()
         ).thenCompose(unused ->
                 // Deploy all registered watches because all components are ready and have registered their listeners.
-                CompletableFuture.allOf(metaStorageMgr.deployWatches(), configurationCatchUpFuture)
+                metaStorageMgr.deployWatches()
         );
 
         assertThat("Partial node was not started", startFuture, willCompleteSuccessfully());
 
-        log.info("Completed recovery on partially started node, last revision applied: " + lastRevision.get()
-                + ", acceptableDifference: " + IgniteSystemProperties.getInteger(CONFIGURATION_CATCH_UP_DIFFERENCE_PROPERTY, 100)
-        );
+        log.info("Completed recovery on partially started node, last revision applied: " + lastRevision.get());
```

Comment:
I would change the "last applied revision" part, and use a different terminology

Reply:
Completed recovery is correct though